repair/row_level.cc

```diff
@@ -1050,7 +1057,9 @@ class repair_meta {
     }
 
     future<> close() noexcept {
-        return _repair_reader.close();
+        return _repair_reader.close().then([this] {
+            return clear_gently();
```

Comment:
How about attach the clear_gently() to stop() instead. close is more about closing the reader while stop is a better place to do destruction.

Reply:
That was my first approach, but then I saw in `row_level_repair::run()` that we auto-close `repair_meta master`, but we don't stop it.  Maybe we should/can, and then it would make more sense to do it in `stop()` instead.

Reply:
done in v3